core/src/main/java/com/sequenceiq/cloudbreak/reactor/DecommissionHandler.java

```diff
@@ -144,6 +153,45 @@ public void accept(Event<DecommissionRequest> event) {
         eventBus.notify(result.selector(), new Event<>(event.getHeaders(), result));
     }
 
+    private DecommissionResult bulkHostsRemoval(DecommissionRequest request, Set<String> hostNames, boolean forced, Stack stack,
+            ClusterDecomissionService clusterDecomissionService, Map<String, InstanceMetaData> hostsToRemove)
+            throws CloudbreakOrchestratorFailedException, CloudbreakException {
+        try {
+            clusterDecomissionService.removeHostsFromCluster(Lists.newArrayList(hostsToRemove.values()));
+        } catch (CloudbreakException e) {
+            if (e.getMessage() != null && e.getMessage().contains("bulk host removal is not supported")) {
+                LOGGER.warn("Bulk host removal was unsuccessful, fallback to single host removal.");
+                return singleHostsRemoval(request, hostNames, forced, stack, clusterDecomissionService, hostsToRemove);
```

Comment:
may worth to throw a specific exception for this case and catch it within different catch block